src/index.js

```diff
@@ -2,6 +2,9 @@ import React from 'react';
 import ReactDOM from 'react-dom';
 import App from './App';
 import * as serviceWorker from './serviceWorker';
+import axios from "axios";
+
+axios.defaults.baseURL = "http://localhost:3000/api/";
```

Comment:
If you are only using the `api` namespace, then communicate with your backend team so that they remove the `v1` namespace

Reply:
Okey, we will fix that!